client/transaction-pool/Cargo.toml

```diff
@@ -41,7 +41,7 @@ sp-consensus = { version = "0.10.0-dev", path = "../../primitives/consensus/comm
 substrate-test-runtime-transaction-pool = { version = "2.0.0", path = "../../test-utils/runtime/transaction-pool" }
 substrate-test-runtime-client = { version = "2.0.0", path = "../../test-utils/runtime/client" }
 sc-block-builder = { version = "0.10.0-dev", path = "../block-builder" }
-codec = { package = "parity-scale-codec", version = "2.0.0" }
+codec = { package = "parity-scale-codec", version = "3.0.0" }
```

Comment:
This is weird, `codec` is already listed as a dependency in this file, so there's no need for this to appear again in `dev-dependencies`.

Reply:
Yeah, in some of the cases they are added with different features, but not here. I try to find the history of it.